pkg/kubelet/volume/cache/actual_state_of_world.go

```diff
@@ -525,14 +542,16 @@ func (asw *actualStateOfWorld) GetMountedVolumesForPod(
 	return mountedVolume
 }
 
-func (asw *actualStateOfWorld) GetAttachedVolumes() []AttachedVolume {
+func (asw *actualStateOfWorld) GetGloballyMountedVolumes() []AttachedVolume {
 	asw.RLock()
 	defer asw.RUnlock()
 	unmountedVolumes := make([]AttachedVolume, 0 /* len */, len(asw.attachedVolumes) /* cap */)
 	for _, volumeObj := range asw.attachedVolumes {
-		unmountedVolumes = append(
-			unmountedVolumes,
-			asw.getAttachedVolume(&volumeObj))
+		if volumeObj.globallyMounted {
+			unmountedVolumes = append(
+				unmountedVolumes,
+				asw.getAttachedVolume(&volumeObj))
```

Comment:
nit: suggest renaming this function to newAttachedVolume, since it just constructs a new object. Can defer to another pr.